channels/ON-CHAIN.md

```diff
@@ -709,35 +709,24 @@ pushed in subsequent transactions. It is up to participants to decide if they
 want to post them at all. Thus the accumulative balances of the accounts in the
 slash transaction can be lower than the channel balance persisted on-chain.
 
-The payload can be either empty or an authenticated by both off-chain state
-transaction.
-
-
-#### Empty payload
-
-If the payload is empty, the last on-chain persisted state and `solo_round` are
-used. In this case the proof of inclusion root hash MUST be equal to the one
-persisted for the channel on-chain. If that state was produced unilaterally,
-i.e. via a `channel_force_progress_tx`, making
-`Channel(channel_id).round != Channel(channel_id).solo_round`, the slash is
-based on the `solo_round` and thus can still be disputed.
-
-- `Channel(channel_id).locked_until := Block.height + Channel(channel_id).lock_period`
-
+The payload must be an authenticated by both off-chain state transaction.
```

Comment:
```suggestion
The payload must be an off-chain state transaction authenticated by both peers.
```

Reply:
This is more or less repeated just below, might be up for removal 🤔

Reply:
Removed